8.2-rc/alpine3.16/zts/Dockerfile

```diff
@@ -163,6 +163,8 @@ RUN set -eux; \
 		--disable-cgi \
 		\
 		--enable-zts \
+# https://externals.io/message/118859
```

Comment:
FYI `--enable-embed` is missing from this Dockerfile, but doesn't look related to this PR.

Reply:
It is only enabled on some variants to minimize the relative size increase (which for alpine based images is significant).

https://github.com/docker-library/php/pull/939#issuecomment-733970326: 
> In coming back to this, I think the question of whether to add this to the CLI variant boils down to the use case of the CLI variants, and whether they'll be substantially affected by a ~14MB increase in size (which in Alpine's case, is almost a 20% size increase).
>
>[...] we start by adding this to the Debian-based CLI variants (so that the Alpine variants are still as small as is reasonably possible)

https://github.com/docker-library/php/pull/1208#discussion_r726497221:
> it's a ~14MB increase. On the Debian-based images, we're on the order of almost 500MB already, so another 14 is pretty negligible in comparison. On Alpine, we're currently at ~84MB, so this would be a pretty noticeable increase in overall image size.

Reply:
Thanks for the background @yosifkit!

Reply:
But are ZTS builds useful for other things than using PHP a library? I mean, this is probably one of the most popular use case.

As we provide Alpine variants in FrankenPHP, having this flag set would allow us (and probably NGINX Unit too) to use the official images for Alpine too.